blueprints/ember-cli-font-awesome-sass/index.js

```diff
@@ -1,11 +1,12 @@
 /*jshint node:true*/
 module.exports = {
   description: '',
-
+  normalizeEntityName: function () {
+  },
```

Comment:
Turns out this `normalizeEntityName` function is critical!  Else my addon cannot run the blueprint from the other project that you're installing this addon in.